src/components/ButtonUndo/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import './button-undo.css';
+const ButtonReset = (props) => {
+    return (
+        <button className="button-undo" onClick={ props.handleClick }>Undo last move</button>
+    )
+}
```

Comment:
All your buttons the same. I think you can create one button and give properties as  `handleClick`, `name`...